packages/useTrie/__tests__/index.test.ts

```diff
@@ -15,12 +15,12 @@ describe('useTrie Hook tests', () => {
     act(() => {
       trieHook.add('abc');
     });
-    expect(trieHook !== result.current).toBe(true);
+    expect(trieHook).not.toBe(result.current);
```

Comment:
It's recommended to use `.not`.

Reply:
This shows that I jumped into using Jest w/o understanding how to use it thoroughly :sweat_smile: